lms/djangoapps/instructor/utils.py

```diff
@@ -185,16 +183,15 @@ def collect_ora2_data(course_id):
 
 
 # pylint: disable=invalid-name
-def ora2_data_queries():
+def ora2_data_queries(include_email, course_id):
     """
     Wraps a raw SQL query which retrieves all ORA2 responses for a course.
     """
-
     RAW_QUERY = """
 SET SESSION group_concat_max_len = 1000000;
 SELECT `sub`.`uuid` AS `submission_uuid`,
 `student`.`item_id` AS `item_id`,
-`student`.`student_id` AS `anonymized_student_id`,
+{0},
```

Comment:
I'd suggest swapping this {0} for a keyword argument substitution like {id_column}.  Then your format call will look like "sql string".format(id_column=id_column), which I think is a little more explicit.  I'm not trying to be nitpicky; I just think query parameterization ought to be as clear as possible.